### ISSUE TITLE
Boost other users volume up to 200%

### ISSUE THREAD

**Issue by koidfas** [maltejur/discord-screenaudio#134]
I known this isn't more important/necessary than the other issues but i think this is a cool default feature to have.

**Comment by Smoukus** [comment]
Yep, I support your suggestion as well. Would be a good QoL change that default Discord already offers.

**Comment by koidfas** [comment]
> Yep, I support your suggestion as well. Would be a good QoL change that default Discord already offers.

sadly, we've been forgotten lol

**Comment by piotr-marendowski** [comment]
Regarding original issue: Well, you could always use `pulseaudio` and `pulsemixer` to achieve it.
